src/pages/index.tsx

```diff
@@ -150,6 +151,22 @@ export default function Inscribete({ location }: PageProps) {
       <Container className="w-11/12 mb-32 lg:w-10/12">
         <VoterStatus />
       </Container>
+      <StickyBanner>
+        <div className="flex flex-col md:flex-row justify-center items-center">
+          <span className="font-bold mb-4 md:mb-0 md:mr-4 block text-lg md:text-lg">
+            Saca tu cita, llama ahora
+          </span>
+          <form action="tel:+1787-777-8682">
+            <Button
+              variant="primary"
+              type="submit"
+              className="h-12 text-md md:text-lg"
+            >
+              (787)-777-8682 Ext. 2362
+            </Button>
+          </form>
```

Comment:
Alguna razón por la que esto es un form y no un anchor tag con `tel:`?

Reply:
Also, crees que debamos marcar el extension automáticamente? 🤔

Reply:
La razon de usar un form y button es para que poder usar los styles del boton ya que el anchor no se parece un boton

Reply:
>Also, crees que debamos marcar el extension automáticamente?

Can we?

Reply:
https://stackoverflow.com/questions/9482633/how-do-i-include-extensions-in-the-tel-uri

Reply:
Si podemos hacer eso estaría nítido pero tenemos que probar que funcione bien.

Reply:
>La razon de usar un form y button es para que poder usar los styles del boton ya que el anchor no se parece un boton

esto no afecta la accesibilidad? creo que tenemos links con button styles...